src/cli.rs

```diff
@@ -311,6 +311,14 @@ fn build_command() -> Command<'static> {
             .hide(true)
             .help("Enable debug mode which does not actually run commands, but returns fake times when the command is 'sleep <time>'.")
         )
+        .arg(
+            Arg::new("stdin-data")
```

Comment:
Can we maybe rename this to `--input`, in analogy to `--output`? Or is there any reason against this?

Reply:
And also change the option such that it works similar to `--output`. Maybe:
```
--input <FROM>
  Control where the input of the benchmark comes from. <FROM> can be:

  null: Read from /dev/null (the default).

  <FILE>: Read the input from the given file
```